bullet_train/app/views/account/invitations/_form.html.erb

```diff
@@ -33,6 +33,10 @@
               </div>
             <% end %>
           <% end %>
+
+          <% with_field_settings form: fields do %>
+            <%= render "account/memberships/fields", membership: fields.object, form: fields %>
+          <% end %>
```

Comment:
This makes it so that new membership attributes are available on the invitation form.